web/src/components/localization-select/localization-select.css

```diff
@@ -0,0 +1,114 @@
+.localization-select {
+    position: relative;
+    font-family: 'Inter', sans-serif;
+
+    &::before {
+        content: '';
+        width: 100%;
+        height: 100%;
+        position: absolute;
+        background: url('./globe.svg') no-repeat left 13px center;
```

Comment:
Curious if we could use the `<GlobeIcon />` component and remove the additional icon code